src/NHibernate/HibernateException.cs

```diff
@@ -12,6 +13,7 @@ namespace NHibernate
 	[Serializable]
 	public class HibernateException : Exception
 	{
+		public List<HibernateException> Exceptions { get; }
```

Comment:
Better follow usual conventions, like the one of the standard [`AggregateException`](https://docs.microsoft.com/en-us/dotnet/api/system.aggregateexception): name this `InnerExceptions`, and expose this as a `ReadOnlyCollection<Exception>`.